scripts/globals/atma.lua

```diff
@@ -163,7 +163,7 @@ function atmaEffectGain(target, effect)
     elseif (pwr == 43) then -- dsp.ki.ATMA_OF_THE_DEEP_DEVOURER
         target:addMod(dsp.mod.SUBTLE_BLOW, 5);
         target:addMod(dsp.mod.STORETP, 5);
-        -- target:addMod(MOD_SONG_CASTING_TIME, -20); Doesnt exist
+        target:addMod(dsp.mod.SONG_SPELLCASTING_TIME, -20);
```

Comment:
One final edit.  Seems like [SONG_SPELLCASTING_TIME](https://github.com/DarkstarProject/darkstar/blob/ab992ecd83cf20dbad3eddd47ec6d745ef4407f0/src/map/utils/battleutils.cpp#L5111) takes positive values as "good" (reduction in cast time).  I double checked: all the 455's item_mods.sql are positive.

Reply:
hahah *cries hysterically*